<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SyncFriendsWidget</class>
 <widget class="QWidget" name="SyncFriendsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>750</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_4">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <item row="3" column="2">
        <layout class="QHBoxLayout" name="horizontalLayout_8">
         <item>
          <widget class="QLineEdit" name="le_acceptFrom"/>
         </item>
         <item>
          <widget class="QPushButton" name="pb_acceptFromAdd">
           <property name="text">
            <string>add</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pb_acceptFromRemove">
           <property name="text">
            <string>remove</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="l_sendFrom">
         <property name="text">
          <string>sendFrom</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="l_sendTo">
         <property name="text">
          <string>sendTo</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QLineEdit" name="le_sendFrom"/>
       </item>
       <item row="1" column="2">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLineEdit" name="le_sendTo"/>
         </item>
         <item>
          <widget class="QPushButton" name="pb_sendToAdd">
           <property name="text">
            <string>add</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pb_sendToRemove">
           <property name="text">
            <string>remove</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="l_acceptFrom">
         <property name="text">
          <string>white list</string>
         </property>
        </widget>
       </item>
       <item row="4" column="2">
        <widget class="QListWidget" name="lw_acceptFrom">
         <property name="selectionMode">
          <enum>QAbstractItemView::MultiSelection</enum>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QListWidget" name="lw_sendTo">
         <property name="selectionMode">
          <enum>QAbstractItemView::MultiSelection</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="pb_sync">
         <property name="text">
          <string>↕ sync</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pb_send">
         <property name="text">
          <string>↑ send</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pb_request">
         <property name="text">
          <string>↓ request</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QGroupBox" name="gb_request">
       <property name="title">
        <string>request</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <widget class="QCheckBox" name="cb_groups">
          <property name="text">
           <string>include groups</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="cb_removeFriendsRequest">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>remove friends</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="gb_localPermissions">
       <property name="title">
        <string>local permissions</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <property name="leftMargin">
         <number>9</number>
        </property>
        <property name="topMargin">
         <number>9</number>
        </property>
        <property name="rightMargin">
         <number>9</number>
        </property>
        <property name="bottomMargin">
         <number>9</number>
        </property>
        <item>
         <widget class="QCheckBox" name="cb_active">
          <property name="text">
           <string>activate</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="cb_removeFriendsActive">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>remove friends</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QPushButton" name="save">
         <property name="text">
          <string>save</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="load">
         <property name="text">
          <string>load</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QListWidget" name="lw_tickets"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QPushButton" name="pb_clearFinish">
         <property name="text">
          <string>remove finished</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pb_dropAllTickets">
         <property name="text">
          <string>remove all</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
